.travis.yml

```diff
@@ -1,9 +1,19 @@
-language: cpp
-sudo: false
+language: python
+python:
+- 3.7-dev
```

Comment:
not critical but still:
- indent nested lists by 2 spaces. also apply for the rest of this file
- run http://www.yamllint.com/ on this file

Reply:
File is intact... Checked that